pyproject.toml

```diff
@@ -83,6 +83,7 @@ bigquery = [
   "pyarrow>=10.0.1",
   "pyarrow-hotfix>=0.4",
   "pydata-google-auth>=1.4.0",
+  "pandas-gbq>=0.26.1",
```

Comment:
Is this required for the parquet loading?

Reply:
Technically, no, but there's a warning thrown without it that tells me to use it.